shell/platform/darwin/ios/framework/Headers/FlutterPlugin.h

```diff
@@ -225,6 +225,42 @@ typedef void (*FlutterPluginRegistrantCallback)(NSObject<FlutterPluginRegistry>*
 - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result;
 @end
 
+#pragma mark -
+/***************************************************************************************************
+ * How the UIGestureRecognizers of a platform view should be blocked.
+ *
+ * UIGestureRecognizers of a platform views can be blocked based on the decisions made by the
+ * Flutter Framework. e.g. When an interact-able widget is covering the platform view and a gesture
+ * happened on the widget, The framework may decide to block the UIGestureRecognizers that are
+ * recognized (via the gesture recognized acton ) on the platform view if any.
+ *
+ * This policy describes how the blocking process is implemented.
+ */
+typedef enum {
+  /**
+   * The flutter framework blocks all the UIGestureRecognizers on the platform view as soon as it
+   * decides they should be blocked.
+   *
+   * If this policy is implemented, only touchesBegan for all the UIGestureRecognizers is guaranteed
+   * to be called.
+   */
+  FlutterPlatformViewGestureRecognizersBlockingPolicyEager,
+  /**
+   * The flutter framework blocks all the UIGestureRecognizers until the `touchesEnded` method is
```

Comment:
Actually I was wrong. The behavior is neither block...until nor block...after.
The gesture recognizers are blocked, but the touchesBegan and touchesEnded callbacks are not. I think we might need to be more implicit here.
What do you think about:
```
The flutter framework waits until the `touchesEneded` method is called for every UIGestureRecognizer on the platform view before blocking the "actions" of all the UIGestureRecognizers.

Reply:
I actually find your original comment easier to process 😄 

Good point about emphasizing that what's being delayed are the actions. Another note is that it's not the "flutter framework" which is blocking the gesture.

How about something like:

```
Flutter blocks the platform view's UIGestureRecognizers from recognizing only after touchesEnded was invoked. This results in the platform view's UIGestureRecognizers seeing the entire touch sequence, but never recognizing the gesture (and never invoking actions).
```